assets/js/theme/global/modal.js

```diff
@@ -1,49 +1,17 @@
-import 'jquery.tabbable';
 import foundation from './foundation';
+import * as focusTrap from 'focus-trap';
```

Comment:
Does import like { createFocusTrap } will make any profit?

Reply:
Your proposal is working, but if you not insist - I prefer using approach from doc
<img width="821" alt="Screenshot 2021-03-01 at 10 13 59" src="https://user-images.githubusercontent.com/66325265/109469731-195aa580-7a77-11eb-8a0f-8892407ba405.png">